src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlException.cs

```diff
@@ -19,14 +19,15 @@ public sealed class SqlException : System.Data.Common.DbException
     {
         private const string OriginalClientConnectionIdKey = "OriginalClientConnectionId";
         private const string RoutingDestinationKey = "RoutingDestination";
+        private const int SqlExceptionHResult = unchecked((int)0x80131904);
```

Comment:
This was moved from HResult.cs so that HRseult.cs could be deleted. This matches the netcore code, too. No functional change.